Test failure in `test_gather_metagenome_downsample`

Test `test_gather_metagenome_downsample` from `tests/test_sourmash.py` is failing in my (Linux) machines: the code checks for
`assert '4.1 Mbp      25.9%    2.4%      NC_011294.1 Salmonella enterica subsp...' in out`
and fails.

Note that there is a match, but with another accession (same species):
`4.1 Mbp      25.9%    2.4%      NC_011274.1 Salmonella enterica subsp...`

Full output:
```
===================================================================================================== FAILURES ===========================================================================================[198/1234]
________________________________________________________________________________________ test_gather_metagenome_downsample _________________________________________________________________________________________

    def test_gather_metagenome_downsample():
        with utils.TempDirectory() as location:
            testdata_glob = utils.get_test_data('gather/GCF*.sig')
            testdata_sigs = glob.glob(testdata_glob)
    
            query_sig = utils.get_test_data('gather/combined.sig')
    
            cmd = ['index', 'gcf_all', '-k', '21']
            cmd.extend(testdata_sigs)
    
            status, out, err = utils.runscript('sourmash', cmd,
                                               in_directory=location)
    
            assert os.path.exists(os.path.join(location, 'gcf_all.sbt.json'))
    
            status, out, err = utils.runscript('sourmash',
                                               ['gather', query_sig, 'gcf_all',
                                                '-k', '21', '--scaled', '100000'],
                                               in_directory=location)
    
            print(out)
            print(err)
    
            assert 'found 11 matches total' in out
            assert 'the recovered matches hit 100.0% of the query' in out
            assert '5.2 Mbp      32.9%  100.0%      NC_003198.1 Salmonella enterica subsp...' in out
>           assert '4.1 Mbp      25.9%    2.4%      NC_011294.1 Salmonella enterica subsp...' in out
E           AssertionError

tests/test_sourmash.py:1849: AssertionError
----------------------------------------------------------------------------------------------- Captured stdout call ----------------------------------------------------------------------------------------------$
running: sourmash in: /tmp/sourmashtest_a5fu841m
arguments ['sourmash', 'index', 'gcf_all', '-k', '21', 'sourmash/tests/test-data/gather/GCF_000009085.1_ASM908v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/$
CF_000006945.2_ASM694v2_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000016785.1_ASM1678v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_00000$
105.1_ASM810v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000018945.1_ASM1894v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000016045.1_AS$
1604v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000009505.1_ASM950v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000011885.1_ASM1188v1_g$
nomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000195995.1_ASM19599v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000008545.1_ASM854v1_genomic.f$
a.gz.sig', 'sourmash/tests/test-data/gather/GCF_000009525.1_ASM952v1_genomic.fna.gz.sig', 'sourmash/tests/test-data/gather/GCF_000007545.1_ASM754v1_genomic.fna.gz.sig'$
running: sourmash in: /tmp/sourmashtest_a5fu841m
arguments ['sourmash', 'gather', 'sourmash/tests/test-data/gather/combined.sig', 'gcf_all', '-k', '21', '--scaled', '100000']

overlap     p_query p_match 
---------   ------- --------
5.2 Mbp      32.9%  100.0%      NC_003198.1 Salmonella enterica subsp...
2.5 Mbp      15.8%  100.0%      NC_011978.1 Thermotoga neapolitana DS...
2.1 Mbp      13.3%   90.5%      NC_009486.1 Thermotoga petrophila RKU...
4.7 Mbp      29.7%   36.2%      NC_003197.2 Salmonella enterica subsp...
1.6 Mbp      10.1%  100.0%      NC_002163.1 Campylobacter jejuni subs...
2.1 Mbp      13.3%   38.1%      NC_000853.1 Thermotoga maritima MSB8 ...
4.1 Mbp      25.9%   17.1%      NC_011294.1 Salmonella enterica subsp...
4.5 Mbp      28.5%   13.3%      NC_006905.1 Salmonella enterica subsp...
4.4 Mbp      27.8%    9.1%      NC_006511.1 Salmonella enterica subsp...
4.4 Mbp      27.8%    6.8%      NC_011080.1 Salmonella enterica subsp...
4.1 Mbp      25.9%    2.4%      NC_011274.1 Salmonella enterica subsp...

found 11 matches total;
the recovered matches hit 100.0% of the query


loaded query: -... (k=21, DNA)
downsampling query from scaled=10000 to 100000
loaded SBT gcf_all

====================================================================================== 1 failed, 267 passed in 20.46 seconds =======================================================================================
```

--- a/tests/test_sourmash.py
+++ b/tests/test_sourmash.py
@@ -1846,7 +1846,8 @@ def test_gather_metagenome_downsample():
         assert 'found 11 matches total' in out
         assert 'the recovered matches hit 100.0% of the query' in out
         assert '5.2 Mbp      32.9%  100.0%      NC_003198.1 Salmonella enterica subsp...' in out
-        assert '4.1 Mbp      25.9%    2.4%      NC_011294.1 Salmonella enterica subsp...' in out
+        assert any(('4.1 Mbp      25.9%    2.4%      NC_011294.1 Salmonella enterica subsp...' in out,
+                    '4.1 Mbp      25.9%    2.4%      NC_011274.1 Salmonella enterica subsp...' in out))
 
 
 def test_gather_save_matches():

